test/collections/DequeParity.good

```diff
@@ -0,0 +1 @@
+SUCCESS
```

Comment:
these tests should be in test/modules/packages/collections e.g. or at a minimum somewhere in test/modules/packages.